parking_permits_app/reversion.py

```diff
@@ -53,7 +53,7 @@ def change_message(self):
                 and target_model.objects.filter(pk=self.new_value).first()
             )
             return f"{self.field.verbose_name}: {old_instance} --> {new_instance}"
-        elif isinstance(self.field, models.DateTimeField):
+        elif isinstance(self.field, models.DateTimeField) and self.old_value:
             format_old = self.old_value.strftime(TIME_FORMAT)
             format_new = self.new_value.strftime(TIME_FORMAT)
```

Comment:
I guess these two lines can be changed to: 

```suggestion
            format_old = self.old_value.strftime(TIME_FORMAT) if self.old_value else ''
            format_new = self.new_value.strftime(TIME_FORMAT) if self.new_value else ''
```

So the result message will be like if the old value is None:

`End time:   --> 2021-11-10 14:00:00`